modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -63,6 +64,10 @@ type HTTPTransport struct {
 	// Maximum number of connections per host. Default: 0 (no limit)
 	MaxConnsPerHost int `json:"max_conns_per_host,omitempty"`
 
+	// Which version of proxy protocol to send when connecting to
+	// an upstream. Default: `don't send proxy protocol`.
+	ProxyProtocolVersion int `json:"proxy_protocol_version,omitempty"`
```

Comment:
I think we should change this to just `ProxyProtocol`, the version is just one of the options when enabling it.

I think we should follow `caddy-l4`'s lead and configure this as a string:

https://github.com/mholt/caddy-l4/blob/2ecee94d269fccbd650a2639c8c8c7bb54c6ec11/modules/l4proxy/proxy.go#L52-L54

Reply:
Yeah, seconding this.